docs/docs-beta/docs/guides/deployment/kubernetes.md

```diff
@@ -1,4 +1,221 @@
 ---
 title: "Deploy to Kubernetes"
 sidebar_position: 21
----
\ No newline at end of file
+---
+
+TODO - INTRODUCTION
+
+## What you'll learn
+
+- How to use Kubernetes and Helm to deploy Dagster
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- Familiarity with [Docker](https://docs.docker.com/)
+- Familiarity with [Kubernetes](https://kubernetes.io/docs/home/)
+- Familiarity with [Helm](https://helm.sh/docs/)
+- A Dagster project to deploy. You can also use the example project [here](/todo). Would be great to have a command they can use to clone the project
+- To have Docker installed. [Docker installation guide](https://docs.docker.com/engine/install/)
+- To have `kubectl` installed. [Kubernetes installation guide](https://kubernetes.io/docs/tasks/tools/)
+- To have a Kubernetes cluster created. If you would like to follow along with this guide on your local machine, you can install Docker Desktop and turn on the included Kubernetes server [Docker Desktop and Kubernetes guide](https://docs.docker.com/desktop/kubernetes/)
+- Access to a Docker image registery, such as Amazon Web Services ECR or DockerHub
+- To have Helm 3 installed. [Helm installation guide](https://helm.sh/docs/intro/install/)
+
+</details>
+
+
+## Step 0a: Understand the Dagster deployment architecture
```

Comment:
The existing guide has a deployment architecture section https://docs.dagster.io/deployment/guides/kubernetes/deploying-with-helm#deployment-architecture

should i do something similar here? If the architecture is more general purpose than just kubernetes, i'd be pro having it as a separate guide/reference page and linking to it as a pre-requisite

Reply:
I'd leave it out of this guide, since we want to focus on the how and less on the what. We can revisit it as another page at some point

Reply:
:relief